docker-compose.yml

```
@@ -28,6 +28,7 @@ services:
       PORT: 8080
       DATABASE_URL: ${DATABASE_URL:-postgres://${POSTGRES_USER:-postgres}:${POSTGRES_PASSWORD}@db:5432/${POSTGRES_DB:-openci}}
       INTERNAL_API_KEY: ${INTERNAL_API_KEY:?INTERNAL_API_KEY environment variable is required}
+      INTERNAL_API_KEY: ${INTERNAL_API_KEY:-}
```

YAMLlint (1.37.1)

[error] duplication of key "INTERNAL_API_KEY" in mapping

(key-duplicates)